modules/programs/broot.nix

```diff
@@ -43,6 +44,14 @@ in {
       '';
     };
 
+    modal = mkOption {
+      default = false;
+      type = types.bool;
+      description = ''
+        Whether to enable modal (vim) mode.
+      '';
+    };
+
```

Comment:
I think this could be a `mkEnableOption`:

```suggestion
    modal = mkEnableOption "modal (vim) mode";
```

Reply:
Thanks for the suggestion!